doc/releases/changelog-0.35.0.md

```diff
@@ -364,30 +492,17 @@
   and the codecov check itself would never execute.
   [(#5101)](https://github.com/PennyLaneAI/pennylane/pull/5101)
 
-* String representations of `ParametrizedHamiltonian` have been updated to match the style of other PL operators.
-  [(#5215)](https://github.com/PennyLaneAI/pennylane/pull/5215)
-
 * `qml.ctrl` called on operators with custom controlled versions will return instances
   of the custom class, and it will also flatten nested controlled operators to a single
   multi-controlled operation. For `PauliX`, `CNOT`, `Toffoli`, and `MultiControlledX`,
   calling `qml.ctrl` will always resolve to the best option in `CNOT`, `Toffoli`, or
   `MultiControlledX` depending on the number of control wires and control values.
   [(#5125)](https://github.com/PennyLaneAI/pennylane/pull/5125/)
```

Comment:
```suggestion
* `qml.ctrl` called on operators with custom controlled versions will now return instances
  of the custom class, and it will flatten nested controlled operators to a single
  multi-controlled operation. For `PauliX`, `CNOT`, `Toffoli`, and `MultiControlledX`,
  calling `qml.ctrl` will always resolve to the best option in `CNOT`, `Toffoli`, or
  `MultiControlledX` depending on the number of control wires and control values.
  [(#5125)](https://github.com/PennyLaneAI/pennylane/pull/5125/)
```